PDK_Abstraction/FinFET14nm_Mock_PDK/Align_primitives.py

```diff
@@ -113,7 +113,7 @@ def gen( pattern, routing):
     gen_lef.json_lef(args.block_name + '.json', args.block_name, cell_pin)
     with open( args.block_name + ".json", "rt") as fp0, \
          open( args.block_name + ".gds.json", 'wt') as fp1:
-        gen_gds_json.translate(args.block_name, '', fp0, fp1, datetime.now())
+        gen_gds_json.translate(args.block_name, '', args.pinSwitch, fp0, fp1, datetime.now())
```

Comment:
Not sure why we have this switch. Is there a situation where we may not want to annotate pin names?